tests/controllers/controller_with_exceptions.py

```diff
@@ -52,3 +54,11 @@ def not_declared_but_thrown(self) -> str:
     @winter.throws(ExceptionWithoutHandler)
     def declared_but_no_handler(self) -> str:
         raise ExceptionWithoutHandler()
+
+    @winter.route_get('redirect/')
+    def redirect(self) -> str:
+        raise RedirectException(redirect_to='/root/')
```

Comment:
Delete this method or ControllerWithRedirectException. Seems like they do the same thing.

Reply:
Done